docs/getting_started/install_manually.md

```diff
@@ -535,9 +535,9 @@ Some systems don't support ACL at all. You will need to set your web server's us
 **Using chown on Linux/BSD/OS X systems that don't support ACL**
 
 ``` bash
-sudo chown -R www-data:www-data var/cache var/logs var/sessions web
-sudo find {var/{cache,logs,sessions},web} -type d | xargs sudo chmod -R 775
-sudo find {app/{cache,logs,sessions},web} -type f | xargs sudo chmod -R 664
+sudo chown -R www-data:www-data var web/var
+sudo find {var,web/var} -type d | xargs sudo chmod -R 775
```

Comment:
Now that options are flat and far less of them, here we can just as well drop the bash expansion syntax in favor of what is supported in plain shell as well:
`sudo find var web/var -type d | xargs sudo chmod -R 775`

Same on 4 more cases below. Do you agree @vidarl? If so we can update INSTALL.md as well in https://github.com/ezsystems/ezplatform/pull/249

Reply:
@DominikaK Update these to `web/var var` now: https://github.com/ezsystems/ezplatform/pull/249